app/components/navpatientheader/MenuOptions/Clinician.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Box, Flex } from 'theme-ui';
+import { useLocation } from 'react-router-dom';
+import { withTranslation } from 'react-i18next';
+
+import { getButtonStyleProps, getFinalSlug } from './menuOptionHelpers';
+
+import Button from '../../elements/Button';
+import viewIcon from '../../../core/icons/viewIcon.svg'
+import profileIcon from '../../../core/icons/profileIcon.svg'
+import uploadIcon from '../../../core/icons/uploadIcon.svg'
+
+const ClinicianMenuOptions = ({ 
+  t, 
+  onViewProfile, 
+  onViewData,
+  onUpload = null,
+}) => {
+  const { pathname } = useLocation();
+  const finalSlug = getFinalSlug(pathname);
+
+  return (
+    <Flex sx={{ ml: 'auto', columnGap: 32 }}>
+      {onViewData && 
+        <Box>
+          <Button
+            id="navPatientHeader_viewDataButton"
+            onClick={onViewData}
+            iconSrc={viewIcon}
+            iconLabel="View"
+            {...getButtonStyleProps(finalSlug === '/data')}
```

Comment:
Same comment as above here - if we're not a fan of this style of abstracting out of the props/attributes, I have no issue with reverting it